acvm_backend_barretenberg/Cargo.toml

```diff
@@ -28,15 +35,18 @@ tempfile = "*"
 
 [features]
 default = ["native"]
+
+std = ["dep:reqwest", "dep:tokio", "dep:futures-util", "dep:dirs", "dep:indicatif"]
```

Comment:
I think we just drop this feature. These impls are going to be behind native/wasm/js feature flags. No reason to keep this.

Reply:
Would you prefer to inline all of these dependencies into `native` and `wasm`? That seems noisy to me and allows for  them to get out of sync.

Reply:
Yeah, I think we should. I'd rather do work to keep the features in sync than to have confusing combinations of features.